app/sessionStore.js

```diff
@@ -59,7 +59,14 @@ module.exports.saveAppState = (payload, isShutdown) => {
         wndPayload.tabs = wndPayload.tabs.filter((tab) => !tab.isPrivate)
       })
     } else {
-      delete payload.perWindowState
+      // Reset the window state except for the ui data
+      payload.perWindowState.forEach((wndPayload) => {
+        Object.keys(wndPayload).forEach((key) => {
+          if (key !== 'ui') {
+            delete wndPayload[key]
```

Comment:
This seems like it's removing all open tabs and frames etc, is this intentional here?